docs/sidebar.yml

```diff
@@ -151,11 +151,6 @@ subsection:
           - page: reference/experimental/cc.md
           - page: reference/experimental/tupled-function.md
       - page: reference/syntax.md
-      - title: Language Versions
-        index: reference/language-versions/language-versions.md
```

Comment:
Is it intended to remove this one? Also, please note that it won’t be removed from the Scala 3 reference documentation (this is why you get [this CI failure](https://github.com/lampepfl/dotty/runs/6354677233?check_suite_focus=true#step:11:636)).

Reply:
you also need to update the file: project/resources/referenceReplacements/static-site-main.html

Reply:
Definitely not intentional. It is an artifact of using git revert. Thanks for catching that.